tests/unit/neptune/new/attributes/series/test_float_series.py

```diff
@@ -51,3 +54,18 @@ def test_log(self):
             values = list(var.fetch_values()["value"].array)
             expected = list(range(0, 5000))
             self.assertEqual(len(set(expected)), len(set(values)))
+
+    def test_float_warnings(self):
+        run = init_run(mode="debug")
+
+        with pytest.warns(NeptuneUnsupportedValue):
+            run["train"].append({"supported_1": 1, "supported_2": 2})
```

Comment:
Let's test multiple values to the same namespaces:
```python
>>> run['f'].extend([1.5, 2.3, str(float("nan")), 4.7])
```
or:
```python
>>> run['b'].extend([str(float("inf")), 5.5, 2.3, 6.7])
```

Reply:
Added 😄